daemon/daemon/policy.go

```diff
@@ -265,7 +266,12 @@ func (d *Daemon) triggerPolicyUpdates(added []uint32) {
 	}
 
 	for _, ep := range d.endpoints {
-		d.regenerateEndpointPolicy(ep, true)
+		err := d.regenerateEndpointPolicy(ep, true)
+		if err != nil {
+			ep.LogStatus(types.Failure, err.Error())
+		} else {
+			ep.LogStatusOK("Policy regenerated")
```

Comment:
This could potentially become very verbose. Let's keep it in but we might have to restrict it later.